.github/workflows/test.yml

```diff
@@ -29,8 +29,3 @@ jobs:
         run: yarn test
         env:
           CI: true
-      - name: dev test
```

Comment:
This also removes the Snowpack dev server tests from every CI run. It was flaky, and slowed down the test suite. But this is now becoming outdated because as of the v3.1 rewrite, all builds are now piped through the dev server internally, so this test either is outdated, or will be soon.